### PR TITLE
Support remaining `target-api-url` commands (-1)

### PR DESCRIPTION
**Pull request by brianaj**
Last part of https://github.com/github/gh-gei/issues/1094 and https://github.ghe.com/github/octoshift/issues/7956

Shipping remaining commands in https://github.ghe.com/github/octoshift/issues/7956#issuecomment-8111321 except `generate-script` as that is more involved and will ship changes in separate PR.

Decided not to functional test since that is more involved and timely and we have shipped enough similar commands that unit testing should be good enough. 

- [ ] Did you write/update appropriate tests
- [ ] Release notes updated (if appropriate)
- [ ] Appropriate logging output
- [ ] Issue linked
- [ ] Docs updated (or issue created)
- [ ] New package licenses are added to `ThirdPartyNotices.txt` (if applicable)

### CODE REVIEW

**Review by gargola — APPROVED** [review]
I'm not familiar with this code but after scanning the changes, nothing obviously concerning popped up, plus; as you said, this is something that has been done several times in the past. LGTM!

File: src/Octoshift/Commands/WaitForMigration/WaitForMigrationCommandBase.cs [hunk 1]
```diff
@@ -27,14 +27,18 @@ public class WaitForMigrationCommandBase : CommandBase<WaitForMigrationCommandAr
     {
         Description = "Personal access token of the GitHub target. Overrides GH_PAT environment variable."
     };
-
+    public Option<string> TargetApiUrl { get; } = new("--target-api-url")
```
Comment (ArinGhazarian) [inline src/Octoshift/Commands/WaitForMigration/WaitForMigrationCommandBase.cs:30]:
```suggestion
    public virtual Option<string> TargetApiUrl { get; } = new("--target-api-url")
```

File: src/Octoshift/Commands/RevokeMigratorRole/RevokeMigratorRoleCommandBase.cs [hunk 2]
```diff
@@ -41,13 +41,22 @@ public override RevokeMigratorRoleCommandHandler BuildHandler(RevokeMigratorRole
...
     }
 
-    public virtual Option<string> GhesApiUrl { get; } = new("--ghes-api-url") { IsRequired = false };
-
+    public virtual Option<string> GhesApiUrl { get; } = new("--ghes-api-url")
+    {
+        IsRequired = false,
+        Description = "The URL of the GitHub Enterprise Server instance, if migrating from GHES. Supports revoking access for exports. Can only configure one of --ghes-api-url or --target-api-url at a time."
+    };
+    public Option<string> TargetApiUrl { get; } = new("--target-api-url")
```
Comment (ArinGhazarian) [inline src/Octoshift/Commands/RevokeMigratorRole/RevokeMigratorRoleCommandBase.cs:55]:
```suggestion
    public virtual Option<string> TargetApiUrl { get; } = new("--target-api-url")
```

**Review by ArinGhazarian — COMMENTED** [review]
Look good, the only thing is the `public virtual` comment. We might have missed it in the previous PRs too.

**Comment by brianaj** [comment]
> Looks good, the only thing is the `public virtual` comment. We might have missed it in the previous PRs too.

okay I have one more PR so will update to virtual for any others missed there 👍🏾

**Review by ArinGhazarian — APPROVED** [review]

**Pull request merged**